docs/modules/SDDs/pages/0033-commodore-component-instance-versioning.adoc

```diff
@@ -0,0 +1,91 @@
+= SDD 0033 - Commodore Component Instance Versioning
+
+:sdd_author:    Aline Abler
+:sdd_owner:     Aline Abler
+:sdd_reviewers: SIG
+:sdd_date:      2025-01-08
+:sdd_status:    draft
+include::partial$meta-info-table.adoc[]
+
+[NOTE]
+.Summary
+====
+This SDD describes possible options to allow specifying different versions for each instance of the same Commodore component.
+It additionally details the options we chose to implement.
+====
+
+== Motivation
+
+With xref:0025-commodore-component-instantiation.adoc[SDD 0025], Commodore can instantiate a component multiple times per cluster if multiple aliases are specified.
+However, all aliases currently use version information from the base component.
+Commodore does not currently support specifying different component versions for each alias.
+
+=== Goals
+
+* Define how different versions for each component instance can be specified and generated.
+
+=== Non-Goals
+
+
+
+== Design Proposal
+
+Commodore will leverage the existing configuration options in `.parameters.components` to retrieve version information for specific instances.
+Instance versions can be specified in `.parameters.components.<instance-name>` and override the value specified in `.parameters.components.<component-name>`.
+Even if all instances provide their own version, a version for the base component (`.parameters.components.<component-name>`) must still be specified.
+
+Component authors must explicitly declare that their component supports multi-versioning.
```

Comment:
We should define the `_metadata` field which indicates this support in this document:

```suggestion
Component authors must explicitly declare that their component supports multi-versioning by setting component parameter `_metadata.multi_versioning=true`.
```